src/ReactiveUI/CreatesCommandBinding.cs

```diff
@@ -4,15 +4,20 @@
 using System.Reactive.Disposables;
 using System.Reactive.Linq;
 using System.Reflection;
-using System.Windows;
 using System.Windows.Input;
 
 #if NETFX_CORE
 using Windows.UI.Xaml;
 using Windows.UI.Xaml.Input;
+#elif WPF
+using System.Windows;
 #endif
 
+#if WPF
+namespace ReactiveUI.Wpf
```

Comment:
I'd be tempted at this point to just pull the WPF copy into this project, and then remove the various `if`s from the shared copy. I think it's a bit neater than redefining the namespace